### PR TITLE
Conform to adapter ns convention

### PR DESCRIPTION
**Pull request by icylisper**

### CODE REVIEW

File: src/pantheon/modules/poynt/adapter/api.clj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-(ns pantheon.adapter.pos.poynt.api
+(ns pantheon.modules.poynt.adapter.api
```
Comment (kriyative) [inline src/pantheon/modules/poynt/adapter/api.clj]: I thought we were dropping `adapter` from the namespaces?

Comment (icylisper) [inline src/pantheon/modules/poynt/adapter/api.clj]: We do have some adaper specific things in `pantheon.modules.{kohls, poynt}.adapter.*`

Comment (kriyative) [inline src/pantheon/modules/poynt/adapter/api.clj]: right, but not the api - that would just be `pantheon.api.poynt`

Comment (icylisper) [inline src/pantheon/modules/poynt/adapter/api.clj]: Ok, makes sense. Updating

**Pull request merged**